CREATE-NEW-SCALER.md

```diff
@@ -8,17 +8,13 @@ In order to develop a scaler, a developer should do the following:
 2. Create the new scaler struct under the `pkg/scalers` folder.
 3. Implement the methods defined in the [scaler interface](#scaler-interface) section.
 4. Create a constructor according to [this](#constructor).
-5. Change the `getScaler` function in `pkg/scaling/scale_handler.go` by adding another switch case that matches your scaler.
+5. Change the `getScaler` function in `pkg/scaling/scale_handler.go` by adding another switch case that matches your scaler. Scalers in the switch are ordered alphabetically, please follow the same pattern.
```

Comment:
Once we have #1025 it will be checked automatically on CI

Reply:
That's nice. My 2 cents: we don't add scalers that often, so I don't think we need to build machinery around it at the moment.

Reply:
But I am not strongly opposed 😄 Just trying to not overcomplicating things, it is harder to maintain then.

Reply:
It's one `awk` and two simple bash pipes so not over-complication 😄I think it's much harder to make every contributor to read every line of contributing guide and even harder to put there everything that has to be done. Btw. I'm happy to help with maintaining it 👍

Reply:
Excellent! 🎉